sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1040,10 +1040,121 @@ def test_sysode_linear_neq_order1_type2():
     for sol in sol1:
         assert checksysodesol(eq1, sol) == (True, [0, 0])
 
+    # Test cases for systems of ODE
+    # https://github.com/sympy/sympy/issues/19763
+    eq11 = (Eq(diff(f(t), t), f(t) + g(t) + 9), Eq(diff(g(t), t), 2*f(t) + 5*g(t) + 23))
+    sol11 = [Eq(f(t), -sqrt(6) * C1 * exp(-sqrt(6) * t + 3 * t) / 2 - C1 * exp(-sqrt(6) * t + 3 * t) - C2 * exp(
+        sqrt(6) * t + 3 * t) + sqrt(6) * C2 * exp(sqrt(6) * t + 3 * t) / 2 - sqrt(6) * exp(
+        -sqrt(6) * t + 3 * t) * Integral(
+        -16 * sqrt(6) * exp(-3 * t + sqrt(6) * t) / 3 + 23 * exp(-3 * t + sqrt(6) * t) / 2, t) / 2 - exp(
+        -sqrt(6) * t + 3 * t) * Integral(
+        -16 * sqrt(6) * exp(-3 * t + sqrt(6) * t) / 3 + 23 * exp(-3 * t + sqrt(6) * t) / 2, t) - exp(
+        sqrt(6) * t + 3 * t) * Integral(
+        23 * exp(-3 * t - sqrt(6) * t) / 2 + 16 * sqrt(6) * exp(-3 * t - sqrt(6) * t) / 3, t) + sqrt(6) * exp(
+        sqrt(6) * t + 3 * t) * Integral(
+        23 * exp(-3 * t - sqrt(6) * t) / 2 + 16 * sqrt(6) * exp(-3 * t - sqrt(6) * t) / 3, t) / 2), Eq(g(t), C1 * exp(
+        -sqrt(6) * t + 3 * t) + C2 * exp(sqrt(6) * t + 3 * t) + exp(-sqrt(6) * t + 3 * t) * Integral(
+        -16 * sqrt(6) * exp(-3 * t + sqrt(6) * t) / 3 + 23 * exp(-3 * t + sqrt(6) * t) / 2, t) + exp(
+        sqrt(6) * t + 3 * t) * Integral(
+        23 * exp(-3 * t - sqrt(6) * t) / 2 + 16 * sqrt(6) * exp(-3 * t - sqrt(6) * t) / 3, t))]
```

Comment:
Updated now. Seems like these spaces came after copying from the Conda prompt.

Reply:
I am not talking just about spaces between operators here, when we paste a solution which is more than one line, it will be better to paste the solutions in a form such that for each dependent variable, the subsequent lines(other than the first one) of the solution for that particular dependent variable have a tab so that a developer can understand where the solution for the dependent variables start and end.

Reply:
The spaces between operators needs to be removed but this particular format that I have described needs to be followed as well for many of the test cases in `test_systems.py`

Reply:
my bad, have formatted now